### PR TITLE
Clear credential manager

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5ae2c35e0c/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0090f240e4/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6ccf81c9e9/how-to/)

**Comment by catlook** [comment]
@cyrichardson need a review pls.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-96e7e50751/how-to/)

File: content/exchange/clear-credential-manager.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+5. Click the arrow next to the entry to display it.
+
+    <img src="{% asset_path exchange/clear-credential-manager/CredentialManager.png %}"/>
+
+6. On the detail page of the credential entry, click **remove** to remove the credential from the manager.
+
+    <img src="{% asset_path exchange/clear-credential-manager/CredentialManagerDetail.png %}"/>
+
+7. Repeat this process for all credentials matching Outlook(15/16) and your email address. There should only be one to three entries like this.
```
Comment (cyrichardson) [inline content/exchange/clear-credential-manager.md]: See my comment above about (15/16)...

Only one to three entries should be available.

File: content/exchange/clear-credential-manager.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+8. Close all windows, and re-open Outlook. When Outlook prompts for your credentials, enter the full email address and password for the mailbox, and check the box **Remember My Credentials**.
```
Comment (cyrichardson) [inline content/exchange/clear-credential-manager.md]: Close all windows and reopen Outlook.

File: content/exchange/clear-credential-manager.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+9. If successful, your Inbox should display, and mail should start syncing.
```
Comment (cyrichardson) [inline content/exchange/clear-credential-manager.md]: If this process is successful, your Inbox should display and mail should start syncing.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5f85a379f5/how-to/).

**Pull request merged**